test/test.js

```diff
@@ -4,9 +4,15 @@ const test = require('../lib/generate-report');
 
 test.generate({
     saveCollectedJSON: true,
-    jsonDir: './test/unit/data/json/',
+    jsonDir: './reports/',
```

Comment:
Please provide a new test file with new data and and a new location so we can generate the report for your specific changes